### ISSUE TITLE
Need more helpful error message in cases when  IMAGE|DIRECTORY or appname is missing.

### ISSUE THREAD

**Issue by sangee2004** [acorn-io/acorn#1457]
Server version - acorn version v0.6.0-132-gdc73d55a+dc73d55a

Steps to reproduce the problem:

```
 acorn run --update  
  ✗  ERROR:  resource name may not be empty

% acorn run --replace 
  ✗  ERROR:  resource name may not be empty
sangeethahariharan@Sangeethas-MBP example1 % 
```

`acorn run` works like `acorn run .` but that is not the case with  `acorn run --update` and` acorn run --replace`.  Why?

In the above cases , a more  helpful error message to indicate that user needs to provide  IMAGE|DIRECTORY if wanting to create a new app or "-n=<app-name> if they want to update existing apps.

**Comment by jsilverio22** [comment]
@sangee2004  this was resolved somewhat by #1445 in that now `acorn run --update` defaults to behavior of `acorn run` so err message should no longer be displayed. moving to QA to confirm this

**Comment by sangee2004** [comment]
Tested with `acorn version v0.6.0-136-gc2c4989d+c2c4989d`

 `acorn run --update` and `acorn run --replace` results in new app being created.
